pom.xml

```diff
@@ -2,11 +2,11 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.ibm.cloud.sdk</groupId>
-    <artifactId>java-sdk-core</artifactId>
+    <groupId>com.ibm.cloud</groupId>
+    <artifactId>sdk-core</artifactId>
     <packaging>jar</packaging>
-    <name>java-sdk-core</name>
-    <version>1.0-SNAPSHOT</version>
+    <name>sdk-core</name>
+    <version>99-SNAPSHOT</version>
```

Comment:
This doesn't really matter since you set it in the Travis script, right?

Reply:
correct, 99-SNAPSHOT would just be the default artifact version # for a non-tagged build